src/_P036_FrameOLED.ino

```diff
@@ -1120,6 +1123,27 @@ boolean Plugin_036(uint8_t function, struct EventStruct *event, String& string)
             # endif // ifdef P036_SEND_EVENTS
             P036_SetDisplayOn(1); //  Save the fact that the display is now ON
           }
+          if ((para1 == F("user")) &&
+            (event->Par2 >= 0) &&
+            (event->Par3 >= 0) &&
+            (event->Par4 >= 0)) {
+            String contrast = parseString(string, 4);
+            String precharge = parseString(string, 5);
+            String comdetect = parseString(string, 5);
+            success = true;
+            P036_data->display->setContrast(contrast.toInt(),precharge.toInt(),comdetect.toInt());
```

Comment:
Why would you create `String` objects using 3 rather costly `parseString()` calls, and parse `int`s from these if the values are already available in `event->Par2..4`? It will be much cheaper, and memory-friendlier, to use `static_cast<some_type>()` these if they don't match according to the compiler.

Reply:
I used the code for the other commands as master copy.
I was not sure if a static cast to the event parameters will work if the user enters non-number characters.

Reply:
Changed!
See my commit [45fe0fe](https://github.com/letscontrolit/ESPEasy/pull/3997/commits/45fe0fe78dbebdb5b1394977e4dc2e9f2cc745d4)